src/components/markdown-renderer/replace-components/mermaid/mermaid-chart.tsx

```diff
@@ -3,10 +3,7 @@ import React, { Fragment, useCallback, useEffect, useRef, useState } from 'react
 import { Alert } from 'react-bootstrap'
 import { useTranslation } from 'react-i18next'
 import { ShowIf } from '../../../common/show-if/show-if'
-
-export interface MermaidChartProps {
-  code: string
-}
+import { MermaidChartProps } from './types'
```

Comment:
Why are those exported in a different file?

Reply:
It was necessary for a short time. Removed